### PR TITLE
Fixed image paths

### PR DESCRIPTION
**Pull request by philBrown**
Congrats on your first PR

### CODE REVIEW

**Comment by philBrown** [comment]
You might also want to [tweak the image sizes](https://www.markdownguide.org/hacks/#image-size) in your `README.md` but I'll leave that to you

**Review by hanahutch — APPROVED** [review]

**Comment by philBrown** [comment]
You'll have to merge this if you want the changes, I don't have permissions (nor should I)

**Review by hanahutch — APPROVED** [review]

**Pull request merged**
